codespan-reporting/src/term.rs

```diff
@@ -9,6 +9,7 @@ use crate::files::Files;
 
 mod config;
 mod renderer;
+mod utils;
```

Comment:
Would it be possible for these to just be local in the `renderer` module? I really try to avoid `util` modules - they end up becoming kitchen sinks hehe :)

Reply:
I'm not so sure that the way I moved this over to render is a good way, rust really seems to want to force it into .. or term/utils/mod.rs :scream: I got it importing from `renderer` but...